docs/mpasseaice.rst

```diff
@@ -24,13 +25,13 @@ typically longer before most analysis is useful::
      config_run_duration = '0002-00-00_00:00:00'
 
 Several streams must be defined in the streams file, typically
-``streams.cice``, (even if they will not be written out --
-``output_interval=="none"``)::
+``streams.seaice`` or ``streams.cice`` in older E3SM runs, (even if they will
+not be written out -- ``output_interval=="none"``)::
```

Comment:
@milenaveneziani, you didn't explicitly request this but I also made changes here and on line 17 above to reference older E3SM runs.